README.md

```diff
@@ -1 +1,14 @@
 # apple-old-version
+An Apple web page clone
+
+
+By: Qoosim AbdulGhaniyy Adekunle
+
+
+This project consists of building an apple website homepage using background
+image and adding gradient to elements. The goal is to make a clone of an old
+version of apple website homepage.
+
+For more information about the project. Check The Odin Project website ["https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-backgrounds-and-gradients"]
+
+For the live version of the project, check here: ["https://raw.githack.com/Qoosim/apple-old-version/feature-1/index.html"]
```

Comment:
You should resolve the conflicts first, they seem to come from the read me file, so maybe pushing the same changes in this branch to the master branch before merging would help you.